modules/styling/lib/cs.ts

```diff
@@ -108,6 +107,18 @@ export function cssVar(input: string, fallback?: string) {
     : `var(${input})`;
 }
 
+/**
+ *
+ * @param key
+ * @returns
+ */
```

Comment:
I found where we implemented this before. I'm not sure how it got removed: https://github.com/Workday/canvas-kit/blob/b775c5737e0019395b32fa18dbd01df9008fca91/modules/react/common/lib/cs.ts#L84-L94

```suggestion
/**
 * Util function to fix an issue with Emotion by
 * appending `EmotionIssue#3066` to end of css variable
 * See issue: [#3066](https://github.com/emotion-js/emotion/issues/3066)
 */
```